lib/beaker/host/windows/exec.rb

```diff
@@ -108,4 +108,14 @@ def selinux_enabled?()
     false
   end
 
+  # Determine if cygwin is actually installed on the SUT. Differs from
+  # is_cygwin?, which is just a type check for a Windows::Host.
+  #
+  # @return [Boolean]
+  def cygwin_installed?
+    output = exec(Beaker::Command.new('cygcheck --check-setup cygwin'), :accept_all_exit_codes => true).stdout 
+    return true if output.match(/cygwin/) and output.match(/OK/)
```

Comment:
So it looks like `cygcheck` always returns `0`, regardless of whether or not it finds a package by the given name.  Oh well....